newdle/client/src/components/UserSearch.js

```diff
@@ -21,16 +24,15 @@ async function searchUsers(data, setResults) {
 }
 
 export default function UserSearch() {
+  const dispatch = useDispatch();
+  const participants = useSelector(getMeetingParticipants);
   const [userModalOpen, setUserModalOpen] = useState(false);
   const [searchResults, setSearchResults] = useState(null);
-  const [participants, setParticipants] = useState([]);
   const [stagedParticipants, setStagedParticipants] = useState([]);
 
-  const removeParticipant = useCallback(
-    participant => {
-      setParticipants(participants.filter(p => p.email !== participant.email));
-    },
-    [participants]
+  const handleRemoveParticipant = useCallback(
```

Comment:
Just curious as I'm not totally familiar with redux hooks, what's the advantage of using `useCallback` here instead of dispatching the action directly?

Reply:
It keeps the identity of `handleRemoveParticipant` consistent between renders so it avoids re-renders of child components it's passed to.

Reply:
In this particular case, would it avoid re-renders of the `<Icon />` component?

Reply:
I actually thoought about the same. And I think I can remove it.